soroban-env-host/src/native_contract/token/public_types.rs

```diff
@@ -1,72 +1,62 @@
 use crate::host::Host;
-use crate::native_contract::base_types::{BigInt, BytesN, Map, Vec};
+use crate::native_contract::base_types::{Bytes, BytesN, Map, Vec};
 use crate::native_contract::token::error::Error;
-use soroban_env_common::{CheckedEnv, TryIntoVal};
+use soroban_env_common::{CheckedEnv, Symbol, TryIntoVal};
 use soroban_native_sdk_macros::contracttype;
 
-pub type U256 = BytesN<32>;
-pub type U512 = BytesN<64>;
-
 #[derive(Clone)]
-#[contracttype]
-pub struct KeyedEd25519Signature {
-    pub public_key: U256,
-    pub signature: U512,
+#[contracttype(lib = "soroban_sdk_auth")]
```

Comment:
Does the native sdk use the lib field?

Reply:
Nope, that was a copy-paste error when moving from soroban-sdk-auth. I'll fix this shortly.